starter-code/pizza.js

```diff
@@ -1 +1,144 @@
 // Write your Pizza Builder JavaScript in this file.
+
```

Comment:
A javascript file typically only has one $(document).ready() function and all the javascript that you will be writing for that file goes within the scope of that single $(document).ready().